### ISSUE TITLE
Tips to use PhotoSwipe with WordPress?

### ISSUE THREAD

**Issue by paaljoachim** [dimsemenov/PhotoSwipe#825]
Hey

PhotoSwipe looks interesting.

I am wondering if you have some documentation on how to use this with WordPress?

Thanks!

**Comment by dimsemenov** [comment]
- There are no specific instructions for WordPress, just like for any other CMS. Instructions on how to include JS/CSS files can be found in WordPress Codex.
- Official WordPress plugin is under development, you may [subscribe here](http://dimsemenov.com/subscribe.html?i=pswp-wp) to get notified about the release. There are already  [some unofficial plugins](https://wordpress.org/plugins/photoswipe-masonry/), you may use them, but I don't control their quality.

**Comment by paaljoachim** [comment]
Hey

I am once again looking into how to add this to WordPress.
What I need is just the initial code for the init file.
I got your Magnific Popup js code to work in WordPress and it would be great to also get Photo Swipe to work as well.  Here is the tutorial I created some time ago:
http://www.easywebdesigntutorials.com/adding-a-lightbox-to-wordpress-without-using-a-plugin/

Here is an example from Colorbox:

```
/* Initialize jQuery Colorbox*/
jQuery(function( $ ){
 $('a[href*=".jpg"], a[href*=".jpeg"], a[href*=".png"], a[href*=".gif"]').colorbox({
    transition:'none',
    rel: 'gallery',
    opacity:.85,
 
    title: function() {
    return $(this).find('img').attr('alt');
    } 
    });
});
```

This means you will have another js plugin shared through my tutorial.
Thanks for helping.
